application/templates/viewdeed.html

```diff
@@ -67,10 +67,16 @@ <h3 class="heading-medium">Effective date and time</h3>
 
         <h3 class="heading-medium">Borrower's digital signature(s)</h3>
         {% for borrower in deed_data.deed.borrowers %}
-          <p class="awaiting-information" id="signature{{ loop.index }}">[Awaiting digital signature
-            of {{ borrower.forename }}
-            {{ borrower.middle_name }}{% if borrower.middle_name %} {% endif %}{{ borrower.surname }}]
-          </p>
+            {% if borrower.signature %}
```

Comment:
Can we sort out the indentation for the added code please. use 2 spaces as well :)